Cesium3DTilesSelection/src/TilesetJsonLoader.cpp

```diff
@@ -788,15 +791,23 @@ TilesetJsonLoader::createLoader(
         }
 
         return TilesetJsonLoader::createLoader(
-            pLogger,
-            pCompletedRequest->url(),
-            tilesetJson);
+                   asyncSystem,
+                   pAssetAccessor,
+                   pLogger,
+                   pCompletedRequest->url(),
+                   requestHeaders,
+                   tilesetJson)
+            .wait();
```

Comment:
We should never call `wait`. Instead, return the Future directly, and change the other places that are returning a value to instead return `asyncSystem.createResolvedFuture(result)`.

Waiting in a `thenInWorkerThread` will monopolize a thread pool / task graph thread, preventing it from doing useful work until the network request completes.